plugins/woocommerce/src/Internal/Admin/Notes/WooSubscriptionsNotes.php

```diff
@@ -75,9 +75,16 @@ public function update_option_woocommerce_helper_data( $old_value, $value ) {
 	}
 
 	/**
-	 * Things to do on admin_init.
+	 * Things to do on admin_head.
 	 */
-	public function admin_init() {
+	public function admin_head() {
```

Comment:
Would be good to have a more meaningful description of the method functionality in the DocBlock, and method name.